packages/manager/apps/pci-rancher/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,23 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsModal } from '@ovhcloud/ods-components/react';
+import React, { useRef } from 'react';
+
+const Modal: React.FC<React.PropsWithChildren & {
+  onCloseCallback: () => void;
+  color: ODS_THEME_COLOR_INTENT;
+}> = ({ color, onCloseCallback, children }) => {
```

Comment:
```suggestion
const Modal: React.FC<React.PropsWithChildren<{
  onClose: () => void;
  color: ODS_THEME_COLOR_INTENT;
}>> = ({ color, onCloseCallback, children }) => {
```

No need to suffix the onClose function with the word `callback`